src/scripts/logBook/LogBookTypes.ts

```diff
@@ -24,4 +24,12 @@ const LogBookTypes: Record<string, LogBookType> = {
         display: 'primary',
         label: 'FOUND',
     },
+    ACHIEVEMENT: {
+        display: 'success',
+        label: '\'CHIEVE',
```

Comment:
I'm not sure about shortening this to `'CHIEVE`

Reply:
`ACHIEVE` or `ACHIEVEMENT`
I wanted to keep the text width same as the other labels